src/queries/volunteer/group.ts

```diff
@@ -91,8 +91,8 @@ export const groupEdit = async ({
   const formData = {
     name,
     description,
-    visibility: visibility ? VISIBILITY_TYPES.ALL : VISIBILITY_TYPES.REGISTERED_ONLY,
-    join_policy: joinPolicy ? JOIN_POLICY_TYPES.OPEN : JOIN_POLICY_TYPES.INVITE_AND_REQUEST,
+    visibility: visibility,
```

Comment:
`visibility` is enough here

Reply:
Yes!
https://github.com/ikuseiGmbH/smart-village-app-app/commit/cc3a3f3cb1db8376e0520d54b44c142b0f4ad913